src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -246,10 +248,35 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
     );
   };
 
+  private handleFilterOutStringClick = (value: string) => {
+    const parentModel = sceneGraph.getAncestor(this, LogsListScene);
+    const lineFilterScene = parentModel.getLineFilterScene();
+    if (lineFilterScene) {
+      lineFilterScene.setState({
+        caseSensitive: true,
+        exclusive: true,
+        regex: false,
+      });
+      lineFilterScene.updateFilter(value, false);
+      reportAppInteraction(
+        USER_EVENTS_PAGES.service_details,
+        USER_EVENTS_ACTIONS.service_details.logs_popover_line_filter,
+        {
+          selectionLength: value.length,
+        }
+      );
+    }
+  };
+
   private handleFilterStringClick = (value: string) => {
     const parentModel = sceneGraph.getAncestor(this, LogsListScene);
     const lineFilterScene = parentModel.getLineFilterScene();
     if (lineFilterScene) {
```

Comment:
Remember to add it as an applied filter, and not populate the input instead. For filter and filter out.

Reply:
Fixed in https://github.com/grafana/explore-logs/pull/963/commits/c1d9c5e8e227b1ec451c69952543e8daf1256781